lib/home/users/manage_users.dart

```diff
@@ -1,68 +1,46 @@
+      builder: (context, snapshot) => DataTable(
+        sortColumnIndex: 0,
+        columns: const [
+          DataColumn(label: Text('Id')),
+          DataColumn(label: Text('Name')),
+          DataColumn(label: Text('Type')),
+          DataColumn(label: Text('Actions'))
+        ],
+        rows: [
+          if (snapshot.data?.docs.reversed case final users?)
+            for (final user in users)
+              DataRow(cells: [
+                DataCell(Text(user['id'])),
+                DataCell(Text(user['name'])),
+                DataCell(Text(user['type'])),
+                const DataCell(Text(''), placeholder: true),
+              ]),
+        ],
       ),
-      body: Padding(
-        padding: const EdgeInsets.fromLTRB(10, 0, 10, 0),
-        child: StreamBuilder<QuerySnapshot>(
-          stream: FirebaseFirestore.instance.collection('users').snapshots(),
-          builder: (context, snapshot) {
-            final List<DataRow> userWidgets = [];
-            if (snapshot.hasData) {
-              final users = snapshot.data?.docs.reversed.toList();
-              for (var user in users!) {
-                final userWidget = DataRow(selected: true, cells: [
-                  DataCell(Text(user['id'])),
-                  DataCell(Text(user['name'])),
-                  DataCell(Text(user['type'])),
-                  const DataCell(Text(''), placeholder: true),
-                ]);
-                userWidgets.add(userWidget);
-              }
-            }
-            return Expanded(
-              child: Scrollbar(
-                child: ListView(
-                  children: [
-                    SingleChildScrollView(
-                      scrollDirection: Axis.horizontal,
-                      child: SingleChildScrollView(
-                        child: DataTable(
-                          // headingTextStyle: const TextStyle(
-                          //   fontSize: 14,
-                          // ),
-                          // dataTextStyle: const TextStyle(
-                          //   fontSize: 12,
-                          // ),
-                          sortColumnIndex: 0,
-                          columns: const [
-                            DataColumn(label: Text('Id')),
-                            DataColumn(label: Text('Name')),
-                            DataColumn(label: Text('Type')),
-                            DataColumn(label: Text('Actions'))
-                          ],
-                          rows: userWidgets,
-                        ),
-                      ),
-                    )
-                  ],
-                ),
-              ),
-            );
-          },
+    );
+
+    return LayoutBuilder(
+      builder: (context, constraints) => SingleChildScrollView(
+        child: SingleChildScrollView(
+          scrollDirection: Axis.horizontal,
+          child: ConstrainedBox(
+            constraints: BoxConstraints(minWidth: constraints.maxWidth),
```

Comment:
The `LayoutBuilder` + `BoxConstraints` stretch out the table for larger screens.